### DISCUSSION TITLE
Babybuddy over docker and caddy

### DISCUSSION THREAD

**Discussion by C-Fu** [babybuddy/babybuddy#652] [Support]
I'm trying to deploy babybuddy over my https://subdomain.domain.com using caddy, but it doesn't seem to work. What kind of env variables should I set?

caddy entry
```
 1 version: "2.1"
 2 services:
 3   babybuddy:
 4     image: lscr.io/linuxserver/babybuddy
 5     container_name: babybuddy
 6     environment:
 7       - TZ=Asia/Kuala_Lumpur
 8       - CSRF_COOKIE_SECURE=False
 9       - CSRF_TRUSTED_ORIGINS=https://subdomain.domain.com,http://hostname:1234,http://ip:1234,http://subdomain.domain.com
10       - SECURE_PROXY_SSL_HEADER=True
11     volumes:
12       - /config:/config
13     ports:
14       - 1234:8000
15     restart: unless-stopped
````

Caddyfile
```
6 }
57 babybuddy.homelab.my {
58         reverse_proxy 192.168.x.x:1234
59 }
```

Browser returns 
This site can’t be reached
subdomain.domain.com took too long to respond.

**Comment by cdubz** [comment]
Tough to say -- there could be lots of things going wrong here and I don't have any experience with Caddy. The error at least sounds like the IP address you have configured Caddy is either wrong or not responding for some reason. Are you able to access Baby Buddy directly from the same IP you're using in the `reverse_proxy` setting?

**Comment by just-grizzle** [comment]
This caddy directive worked for me:

```
sub.domain.tld {
        reverse_proxy http://144.24.xx.yy:8002
}
```

Can't tell if it's the scheme or that I'm referring a public IP.  Also I struggled with timeouts from Caddy - forgot to open Port in the firewall. 

Maybe this helps you.
